index.html

```diff
@@ -3,8 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <title>Tech It Easy Dashboard</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <div id="to-sell"></div>
+    <div id="sales-target"></div>
+    <div id="actual-sales"></div>
+    <div id="hitachi-tv"></div>
+    <div id="philips-tv"></div>
+    <div id="tv-generator"></div>
 
+    <button onclick="sortTvOnPrice(inventory)">Sort on Price</button>
+    <button onclick="ambiLightTV(inventory)"> AmbiLight Tv</button>
+    <button onclick="soldOutTv(inventory)">Sold Out</button>
+
+    <script src="main.js"></script>
 </body>
 </html>
```

Comment:
looks good! Ik had de buttons ongeveer hetzelfde, maar bij mij doen ze ook niks. In de uitwerking van de opdracht overigens ook niet, dus I guess dat het zo goed is!